lib/fsm/follow.go

```diff
@@ -65,60 +66,50 @@ func (e *PlanRolledBackEvent) isTerminalEvent() bool { return true }
 func FollowOperationPlan(ctx context.Context, getPlan GetPlanFunc) <-chan PlanEvent {
 	ch := make(chan PlanEvent, 100)
 	// Send an initial batch of events from the initial state of the plan.
-	plan, err := getPlan()
-	if err != nil {
-		logrus.WithError(err).Error("Failed to load plan.")
-	}
-	if plan != nil {
```

Comment:
This is a minor deviation from the existing behavior but I have not found evidence where both `plan` and `err` could be `nil`.